### PR TITLE
Add better message for jwtSecret and rememberMe keys with `jhipster info`

### PR DESCRIPTION
**Pull request by mraible**
A better alternative might be to delete the keys altogether since they'll be regenerated.

### CODE REVIEW

**Comment by mraible** [comment]
@jhipster/developers What do you think about this change? Deleting the keys might be even better since they aren't needed to regenerate an app.

**Comment by mshima** [comment]
Removing should be the way imo.
```
console.log(JSON.stringify({ …this.jhipsterConfig, jwtSecretKey: undefined, rememberKey: undefined }, null, 2));
```

File: generators/info/index.js [hunk 1]
```diff
@@ -60,14 +60,8 @@ module.exports = class extends BaseGenerator {
...
         let result = shelljs.cat('.yo-rc.json');
-        result = result.replace(
-          /"rememberMeKey": ".*"/g,
-          '"rememberMeKey": "YourJWTSecretKeyWasReplacedByThisMeaninglessTextByTheJHipsterInfoCommandForObviousSecurityReasons"'
-        );
-        result = result.replace(
-          /"jwtSecretKey": ".*"/g,
-          '"jwtSecretKey": "YourJWTSecretKeyWasReplacedByThisMeaninglessTextByTheJHipsterInfoCommandForObviousSecurityReasons"'
-        );
+        result = result.replace(/"rememberMeKey": ".*"/g, '"rememberMeKey": "Delete this line so this value is regenerated"');
```
Comment (mraible) [inline generators/info/index.js:63]:
```suggestion
    
```

File: generators/info/index.js [hunk 2]
```diff
@@ -60,14 +60,8 @@ module.exports = class extends BaseGenerator {
...
+        result = result.replace(/"jwtSecretKey": ".*"/g, '"jwtSecretKey": "Delete this line so this value is regenerated"');
```
Comment (mraible) [inline generators/info/index.js:64]:
```suggestion
        const result = JSON.stringify({ …this.jhipsterConfig, jwtSecretKey: undefined, rememberMeKey: undefined }, null, 2);
```

File: generators/info/index.js [hunk 3]
```diff
@@ -60,14 +60,8 @@ module.exports = class extends BaseGenerator {
       displayConfiguration() {
         const done = this.async();
         let result = shelljs.cat('.yo-rc.json');
```
Comment (mraible) [inline generators/info/index.js:62]:
```suggestion
```

File: generators/info/index.js [hunk 4]
```diff
@@ -60,14 +60,8 @@ module.exports = class extends BaseGenerator {
...
-        result = result.replace(
-          /"rememberMeKey": ".*"/g,
-          '"rememberMeKey": "YourJWTSecretKeyWasReplacedByThisMeaninglessTextByTheJHipsterInfoCommandForObviousSecurityReasons"'
-        );
-        result = result.replace(
-          /"jwtSecretKey": ".*"/g,
-          '"jwtSecretKey": "YourJWTSecretKeyWasReplacedByThisMeaninglessTextByTheJHipsterInfoCommandForObviousSecurityReasons"'
-        );
+
```
Comment (mraible) [inline generators/info/index.js:63]:
```suggestion
```

File: generators/info/index.js [hunk 5]
```diff
@@ -58,28 +58,16 @@ module.exports = class extends BaseGenerator {
...
-        let result = shelljs.cat('.yo-rc.json');
-        result = result.replace(
-          /"rememberMeKey": ".*"/g,
-          '"rememberMeKey": "YourJWTSecretKeyWasReplacedByThisMeaninglessTextByTheJHipsterInfoCommandForObviousSecurityReasons"'
-        );
-        result = result.replace(
-          /"jwtSecretKey": ".*"/g,
-          '"jwtSecretKey": "YourJWTSecretKeyWasReplacedByThisMeaninglessTextByTheJHipsterInfoCommandForObviousSecurityReasons"'
-        );
+        const result = JSON.stringify({ ...this.jhipsterConfig, jwtSecretKey: undefined, rememberMeKey: undefined }, null, 2);
```
Comment (mshima) [inline generators/info/index.js:61]:
```suggestion
        // Omit sensitive information.
        const result = JSON.stringify({ ...this.jhipsterConfig, jwtSecretKey: undefined, rememberMeKey: undefined }, null, 2);
```

**Review by mshima — COMMENTED** [review]
Looks good to me if it's omitting the sensitive fields correctly.

**Pull request merged**